lib/galaxy/tools/evaluation.py

```diff
@@ -639,6 +640,9 @@ def _build_environment_variables(self):
                 else:
                     environment_variable_template = ""
                 is_template = False
+            elif inject.startswith("oidc_"):
```

Comment:
```suggestion
            elif inject and inject.startswith("oidc_"):
```
since `inject` can be `None`